tests/h/services/group_test.py

```diff
@@ -5,344 +5,408 @@
 import mock
 import pytest
 
-from h.models import Group, GroupScope
+from h.models import Group, User, GroupScope
 from h.models.group import JoinableBy, ReadableBy, WriteableBy
 from h.services.group import GroupService
 from h.services.group import groups_factory
-from h.services.organization import OrganizationService
-
+from h.services.user import UserService
 from tests.common.matchers import Matcher
 
 
-class TestGroupService(object):
-    def test_create_private_group_returns_group(self, service):
-        group = service.create_private_group('Anteater fans', 'cazimir')
+class TestGroupServiceCreatePrivateGroup(object):
+    def test_create_private_group_returns_group_model(self, creator, svc):
+        group = svc.create_private_group('Anteater fans', creator.userid)
 
         assert isinstance(group, Group)
 
-    def test_create_private_group_sets_group_name(self, service):
-        group = service.create_private_group('Anteater fans', 'cazimir')
+    def test_create_private_group_sets_group_name(self, creator, svc):
+        group = svc.create_private_group('Anteater fans', creator.userid)
 
         assert group.name == 'Anteater fans'
 
-    def test_create_private_group_sets_group_authority(self, service):
-        group = service.create_private_group('Anteater fans', 'cazimir')
+    def test_create_private_group_sets_group_authority(self, svc, creator, pyramid_request):
+        group = svc.create_private_group('Anteater fans', creator.userid)
 
-        assert group.authority == 'example.com'
+        assert group.authority == pyramid_request.authority
 
-    def test_create_private_group_sets_group_creator(self, service, users):
-        group = service.create_private_group('Anteater fans', 'cazimir')
+    def test_create_private_group_sets_group_creator(self, svc, creator):
+        group = svc.create_private_group('Anteater fans', creator.userid)
 
-        assert group.creator == users['cazimir']
+        assert group.creator == creator
 
-    def test_create_private_group_sets_description_when_present(self, service):
-        group = service.create_private_group('Anteater fans', 'cazimir', 'all about ant eaters')
+    def test_create_private_group_sets_description_when_present(self, svc, creator):
+        group = svc.create_private_group('Anteater fans', creator.userid, 'all about ant eaters')
 
         assert group.description == 'all about ant eaters'
```

Comment:
Just a comment on the pre-existing code: I probably would have put all these asserts (about the different properties of the created group) in one test, rather than a test each